src/controllers/action/get-all.js

```diff
@@ -0,0 +1,18 @@
+const HttpStatus = require('http-status-codes');
+const boom = require('boom');
+const logger = require('../../lib/logger');
+
+const {
+    Action,
+    User,
+} = require('../../models');
+
+module.exports = (req, res, next) => {
+    Action.find({ userId: req.userData.userId }).exec()
+        .then(result =>
```

Comment:
i think it's dangerous that you return straight value from database - for example i think that in this place stats will be not required to return